### ISSUE TITLE
StripChromeSymbols.py does not function under python 3, ONLY when launched from UIforETW

### ISSUE THREAD

**Issue by ariccio** [google/UIforETW#13]
This ticket is partly so that I can track the issue.

StripChromeSymbols.py, as called from CUIforETWDlg::StripChromeSymbols fails under Python 3.

The failing output is:

```
> xperf -i "C:\Users\Alexander Riccio\Documents\etwtraces\2015-04-23_02-28-21_AlexanderRiccio.etl" -tle -tti -a symcache -dbgid
Traceback (most recent call last):
  File "C:\Users\Alexander Riccio\Documents\GitHub\UIforETW\UIforETW\..\bin\StripChromeSymbols.py", line 177, in <module>
    main( )
  File "C:\Users\Alexander Riccio\Documents\GitHub\UIforETW\UIforETW\..\bin\StripChromeSymbols.py", line 104, in main
    for line in os.popen(command).readlines():
  File "C:\Python34\lib\os.py", line 943, in popen
    bufsize=buffering)
  File "C:\Python34\lib\subprocess.py", line 823, in __init__
    errread, errwrite) = self._get_handles(stdin, stdout, stderr)
  File "C:\Python34\lib\subprocess.py", line 1005, in _get_handles
    p2cread = _winapi.GetStdHandle(_winapi.STD_INPUT_HANDLE)
OSError: [WinError 6] The handle is invalid
```

I've confirmed that the script runs properly on it's own - I commented `child.Run` out, and pasted the command into a separate command prompt, and the script ran correctly - but when run programmatically from UIforETW, it fails.

I also tried replacing StripChromeSymbols.py with a version that did nothing other than try to call `"echo \"text n crap\""` via `subprocess.getoutput`, and that too failed from UIforETW. 

This behavior is not documented anywhere, and is certainly surprising.

My best guess is that it has something to do with HANDLE inheritance.

**Comment by ariccio** [comment]
Oops, I got a bit ahead of myself. One commit/PR actually.

I forgot that I'd not yet submitted a PR for Python 3 support in StripChromeSymbols.py

**Comment by randomascii** [comment]
Resolved by a01b389 (set the stdinput handle) and 1c00b48 (fix dict.keys problem).
